tsup.config.ts

```diff
@@ -0,0 +1,80 @@
+import {globSync} from 'glob';
+import {print} from 'q-i';
+import {join} from 'path';
+import { defineConfig } from 'tsup';
+
+
+const RESOURCES_PATH = 'src/main/resources';
+const ASSETS_PATH = `${RESOURCES_PATH}/assets`;
+const CLIENT_GLOB_EXTENSIONS = '{tsx,ts,jsx,js}';
+const SERVER_GLOB_EXTENSIONS = '{ts,js}';
+
+const CLIENT_FILES = globSync(`${ASSETS_PATH}/**/*.${CLIENT_GLOB_EXTENSIONS}`);
+// print(CLIENT_FILES, { maxItems: Infinity });
+
+const SERVER_FILES = globSync(
+	`${RESOURCES_PATH}/**/*.${SERVER_GLOB_EXTENSIONS}`,
+	{
+		absolute: false,
+		ignore: globSync(`${ASSETS_PATH}/**/*.${SERVER_GLOB_EXTENSIONS}`)
+	}
+);
```

Comment:
tsup does not accept windows-like paths
```suggestion
).map(dir => dir.replace(/\\/g,'/'));
```